src/main/flight/gps_rescue.c

```diff
@@ -567,8 +567,8 @@ void updateGPSRescueState(void)
         // Only allow new altitude and new speed to be equal or lower than the current values (to prevent parabolic movement on overshoot)
         const int32_t newAlt = (lineSlope * rescueState.sensor.distanceToHomeM + lineOffsetM) * 100;
         
-        if (rescueState.sensor.distanceToHomeM <= gpsRescueConfig()->slowdownDistanceM) {
-            newSpeed = gpsRescueConfig()->rescueGroundspeed * rescueState.sensor.distanceToHomeM / gpsRescueConfig()->slowdownDistanceM;
+        if (rescueState.sensor.distanceToHomeM <= GPS_RESCUE_SLOWDOWN_DISTANCE_M) {
```

Comment:
And maybe a comment here describing how the "slow down" logic works.

Reply:
Done